specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -5896,9 +5896,32 @@
             "modelAsString": true
           }
         },
+        "partnerTopicReadinessState": {
+          "description": "The readiness state of the corresponding partner topic.",
+          "enum": [
+            "NotActivatedByUserYet",
+            "ActivatedByUser",
+            "DeactivatedByUser",
```

Comment:
>DeactivatedByUser [](start = 13, length = 17)

Are there an difference b/w this and NotActivated?

Reply:
This corresponds to the mirrored partner topic state of being never activated yet and deactivated. The first case represents that the event channel was created by partner which in turns results in creating the partner topic in the user context. this topic needs to be activated by user within certain amount of time (7 days is the default if not specified by publisher0 otherwise it will be deleted.

This deletion behavior only happen if the topic is never activated ever to ensure the user is not surprised by such creation. But customer can come and activate the topic and later deactivate it. At this point, he is aware of the topic resource and no need to auto delete it